### PR TITLE
Added NullCon 2020

### PR DESCRIPTION
**Pull request by aravindputrevu**
**Details of the conference**: NullCon

- Website: https://nullcon.net/website/goa-2020/cfp.php
- Date: 6th March 2020
- Venue: Taj Hotel and Convention Center, Dona Paula
- Description: Nullcon is an annual security conference held in Goa, India. The focus of the conference is to showcase the next generation of offensive and defensive security technology. We happily open doors to researchers and hackers around the world and the universe, working on the next big thing in security and request everyone to submit their new research.

### CODE REVIEW

File: year-pages/2020.md [hunk 1]
```diff
@@ -10,3 +10,4 @@
...
 | [Machine Learning Developers Summit](https://www.mlds.analyticsindiasummit.com/) | 22-23 Jan | NIMHANS Convention Center, Bengaluru | India’s leading Machine Learning innovators and practitioners to share their ideas and experience about machine learning tools, advanced development & developer products | No |
 | [India OS](https://indiaos.in/) | 18 Jan | Zerodha, Bengaluru, Karnataka | See what's shaping the future of software, join a community connected by code and turn ideas into innovations. The goal is to provide software developers and policy makers, communities and enthusiasts a platform to meet and interact. | Free to attend |
+| [NullCon](https://nullcon.net/website/goa-2020/cfp.php) | 06 March | Taj Hotel and Convention Center, Dona Paula | Nullcon is an annual security conference held in Goa, India. The focus of the conference is to showcase the next generation of offensive and defensive security technology. We happily open doors to researchers and hackers around the world and the universe, working on the next big thing in security and request everyone to submit their new research. | No |
```
Comment (nikhita) [inline year-pages/2020.md]:
```suggestion
| [NullCon](https://nullcon.net/website/goa-2020/cfp.php) | 06 Mar | Taj Hotel and Convention Center, Dona Paula | Nullcon is an annual security conference held in Goa, India. The focus of the conference is to showcase the next generation of offensive and defensive security technology. | No |
```

Comment (nikhita) [reply year-pages/2020.md]: Oh, also please move this conference near the top of the README to maintain the order. Thanks!

Comment (aravindputrevu) [reply year-pages/2020.md]: Sure.

**Comment by aravindputrevu** [comment]
@nikhita Made the requested changes.

**Comment by nikhita** [comment]
Please also remove the line -  `We happily open doors to researchers and hackers around the world and the universe, working on the next big thing in security and request everyone to submit their new research.` - to keep the description short and concise.

And this conf needs to be between the April and Feb conferences. :smile:

**Pull request merged**